protocols/gossipsub/src/behaviour.rs

```diff
@@ -332,6 +335,9 @@ pub struct Behaviour<D = IdentityTransform, F = AllowAllSubscriptionFilter> {
 
     /// Keep track of a set of internal metrics relating to gossipsub.
     metrics: Option<Metrics>,
+
+    /// Connection handler message queue channels.
+    handler_send_queues: HashMap<PeerId, RpcSender>,
```

Comment:
> because `ConnectionHandler`s may send to different `PeerId`s during its existence right?

No. A `ConnectionHandler` only ever connects the local node to a single remote node (i.e. remote `PeerId`) during its existence.

I think the problem that Thomas is hinting at, is that we might have multiple connections to a single `PeerId`. In other words, there might be multiple `RpcSender`s per `PeerId`.